pkg/metrics/metrics.go

```diff
@@ -0,0 +1,26 @@
+package metrics
```

Comment:
shall we merge this file with store.go into type.go?  Put them together can get better readability.